<launch>
  <arg name="video_device" default="0" />
  <arg name="image_width" default="640" />
  <arg name="image_height" default="480" />
  <arg name="camera_frame_id" default="head_camera" />
  <arg name="image" default="/jetson_nano/image_raw"/>
  <node name="camera_node" pkg="jetson_cam" type="camera_node" output="screen">
    <param name="video_device" value="$(arg video_device)" />
    <param name="image_width" value="$(arg image_width)" />
    <param name="image_height" value="$(arg image_height)" />
    <param name="image" value="$(arg image)"/>
    <param name="camera_frame_id" value="$(arg camera_frame_id)" />
  </node>
</launch>